R/auto-test.R

```diff
@@ -113,6 +115,15 @@ auto_test_package <- function(pkg = ".", reporter = default_reporter(), hash = T
         devtools::r_env_vars(),
         test_dir(test_path, env = env, reporter = reporter$clone(deep = TRUE))
       )
+    } else if (length(helper) > 0) {
```

Comment:
Is this this same code as the block above? If so, I think a simpler solution would be to add `helper` to the `code` vector.

Reply:
Almost all: the "active" code is exactly the same; the message changed, reporting "Changed helper: " instead of "Changed code: ". If that is not a relevant "feature", your solution seems better for me too.

I have implemented your purpose, so you can choose what you prefer to commit.
(I have also added a little "why" above the update of `test` and `code`)